v1/endpoints.md

```diff
@@ -942,6 +948,213 @@ See [RFC 6266](http://tools.ietf.org/html/rfc6266) (description of the Content-D
 
 ---
 
+# Access Group Endpoints
+
+## Fetch access groups endpoint
+
+Retrieve the access groups, if any.
+
+### Request
+
+```
+GET /access-group?page=1&pageSize=10 HTTP/1.1
+Accept: application/json
+Authorization: Bearer {token}
+```
+
+- `page` (optional)
+  - specifies the page to fetch; 1-based
+  - defaults to 1
+- `pageSize` (optional)
+  - specifies the page size
+  - defaults to 10
+  - maximum allowed value is 1000
+
+### Response
+
+```
+200 OK
+Content-type: application/json
+
+{
+    "data": [
+        {
+            attribute: value,
+            ...
+        },
+        ...
+    ],
+    "hasMore": boolean,
+    "page": integer,
+    "pageSize": integer
+}
+```
+
+- `data`
+  - contains a (potentially empty) list of access groups
+- `hasMore`
+  - specifies whether more pages exist
+- `page`
+  - the requested page
+- `pageSize`
+  - the requested page size
+
+Response codes:
+- `200 OK`
+- `400 Bad request`
+
+---
+
+## Fetch access group by ID endpoint
+
+Retrieves the access group with the specified ID, if any.
+
+### Request
+
+```
+GET /access-group/{id} HTTP/1.1
+Accept: application/json
+Authorization: Bearer {token}
+```
+
+### Response
+
+```
+200 OK
+Content-type: application/json
+
+{
+    "data": {
+        attribute: value,
+        ...
+    }
+}
+```
+
+- `data`
+  - contains the (potentially empty) resource with the specified ID
+  - at most 10 nested resources per type will be returned; more information on retrieving additional resources can be found in `__resources`
+
+Response codes:
+- `200 OK`
+- `400 Bad request`
+- `403 Unauthorized`
```

Comment:
Haven't read every corner of the docs yet, so my following question might already be answered elsewhere in the docs as a general rule of thumb or similar..

Would trying to fetch a non-exising access group also result in 403?

Reply:
Yes, it will :)

Reply:
Cool 👍 Assuming that is to not leak data by disclosing what exists or not... I've been surprised by that kind of behaviour before being a client, is it described as a heads-up anywhere?

Reply:
It somewhat is in the beginning of endpoints.md, but I agree with you that we can expand it to be more explicit.

As a rule of thumb with this API you will get 404 if you hit a non-existing endpoint and a 403 if you request a non-existing resource or one you do not have access to.